backend/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# --- Stage 1 ---
+# Base Layer, contains Go build tools
+# Name this stage 'builder' so we can refer to it later
+FROM golang:alpine AS builder
+
+# Enable Go modules
+ENV GO111MODULE=on
+
+# Copy files to image and set current
+# working directory to /app/src
+COPY . /app/src
+WORKDIR /app/src
```

Comment:
You can avoid running `go mod download` (which can take a while as your dependency graph grows) on every build by splitting the copies and allowing docker to cache the download step separately from the rest of your code:

https://sourcegraph.com/github.com/ubclaunchpad/inertia/-/blob/Dockerfile#L10-21